kythe-verification/testdata/basic/ImportsRef.hs

```diff
@@ -0,0 +1,8 @@
+-- Verify the references to the imported modules
```

Comment:
Done.

But I'm failing to add vname verification on base:Prelude import. It's not crucial as there's no visible token to bind it, but it would be nice to assert it.